### DISCUSSION TITLE
playlist results problems

### DISCUSSION THREAD

**Discussion by error-reporting** [deniscerri/ytdlnis#603] [General]
is weird that ytdlnis 1.8.0 acts differently playlist to see the diff: https://m.youtube.com/playlist?list=PLdhrcCVXurgLpJ1nslCxOdsYgh4xo8WqD
app ver 1.7.9.2 shows the playlist title properly
and app ver 1.8.0 shows the playlist title as watch

**Comment by zaednasr** [comment]
looking into it

**Comment by zaednasr** [comment]
Alright i fixed it

**Comment by error-reporting** [comment]
i do not see any change on my side when using yt-dlp data extractor
your fix are not seen at my side
![Uploading Screenshot_20241104-085643.jpg…]()

**Comment by error-reporting** [comment]
do i need to reinstall the app to see the fix take effect

**Comment by zaednasr** [comment]
Well when i release the new version is fixed
If you want dm me on telegram to get it rn

**Reply by STOIC5007** [reply]
What's your telegram channel

**Reply by error-reporting** [reply]
hey @STOIC5007 that question you ask me or zaednasr

**Reply by STOIC5007** [reply]
Zaednasr since he said he has the fixed version and will send it to you through he's channel however you can help me get it aswell That's if he's already sent it to you then i could get it from you?

**Reply by error-reporting** [reply]
no zaednasr have not sent that version yet i just wait for update like other ytdlnis user

**Reply by error-reporting** [reply]
STOIC5007 do you see the same bug as in disscusion #603 that i reported

**Reply by STOIC5007** [reply]
Yes in version 1.8 which meant i had to downgrade back to 1.7.9 yet i keep encountering a problem of my own which also led in me noticing the error you reported. Each time i download items in a playlist there's a 50/50 chance ill get an error to upgrade to the latest python. Has that happened to you if so how did you deal with it?

**Reply by error-reporting** [reply]
@STOIC5007 if a playlist affected by the bug then any video in that affected playlist even when shared as single video is also affected by the bug too and some single video that is not in a playlist are also affected by the bug
the answer for your first question is: depending on your current yt-dlp versions
the answer for your second question is: i deal with the #603 bug mostly by using newpipe data extractor in ytdlnis setting as for the upgrade to latest Python versions error: update to ytdlnis 1.8.0

**Reply by error-reporting** [reply]
hey STOIC5007 are you sleeping

**Reply by error-reporting** [reply]
STOIC5007 where are you

**Comment by error-reporting** [comment]
@STOIC5007 where are you

**Reply by zaednasr** [reply]
I'll try to push this update asap

**Reply by zaednasr** [reply]
But too many feature requests in my list. The app is a mess rn

**Reply by error-reporting** [reply]
that question is for @STOIC5007 not you @zaednasr
